content/rv/administration/setup_and_editing/view-edit-subscription.md

```diff
@@ -41,23 +41,20 @@ databases.
 ## Defining Access to your Subscription
 
 After you create a Redis Enterprise VPC (RV) subscription, you can configure VPC
-peering and a CIDR whitelist to allow more direct access to your VPC. VPC
-peering and CIDR whitelists are capabilities provided by [Amazon Virtual Private Cloud]
-(https://docs.aws.amazon.com/vpc/latest/userguide/) (Amazon VPC), but you can
-configure them here in RV.
+peering and a CIDR whitelist to allow more secure access to your VPC.
 
 - [VPC peering](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html) -
-    The route tables of each VPC have access to the entire CIDR block of the peered VPC.
+    Connect two VPCs together, allowing network access between them.
 - [CIDR whitelist](https://docs.aws.amazon.com/vpc/latest/peering/peering-configurations-partial-access.html) -
-    The route table of your VPC has access to the specified CIDR blocks and security groups.
+    narrow down the access to your entire subscription by specifing the relevant CIDR blocks and security groups.
```

Comment:
Limit network access between the VPCs to the specified CIDR blocks and security groups.

Reply:
Sounds good